.bazelrc

```diff
@@ -0,0 +1,9 @@
+build --action_env CC=clang
```

Comment:
Would using an LLVM toolchain from here be a more hermetic alternative (i.e., not require the user to install clang manually)?
https://github.com/grailbio/bazel-toolchain

I used it locally and it seemed to work.

Reply:
IMO using host clang is preferred in cases where we build a cmake version with the same clang binary.